repository-data/webfiles/src/main/resources/site/freemarker/common/location-article.ftl

```diff
@@ -0,0 +1,433 @@
+<#assign hasDunsNumber = document.dunsnumber?? && document.dunsnumber?has_content />
+<#assign hasOdsCode = document.odscode?? && document.odscode?has_content/>
+<#assign hasOtherLocationData = hasUniquePropertyRefrenceNumber || hasDunsNumber || hasOdsCode />
+<#assign hasDirectionToSiteByCar = document.directiontositebycars?? && document.directiontositebycars?has_content/>
+<#if hasLocalAddress >
+<#assign hasGeocoordinates = document.locAddress.geocoordinates?? && document.locAddress.geocoordinates.content?has_content />
+<#else>
+<#assign hasGeocoordinates = hasLocalAddress />
+</#if>
+
+<article class="article article--general">
+    <div class="grid-wrapper grid-wrapper--full-width grid-wrapper--wide" aria-label="Document Header">
+        <div class="local-header article-header article-header--detailed">
+            <div class="grid-wrapper">
+                <div class="article-header__inner">
+                    <h1 class="local-header__title" data-uipath="document.title" itemprop="name">${document.name}</h1>
+                    <hr class="hr hr--short hr--light">
+
+                    <div class="article-header__subtitle" data-uipath="article.description"><span itemprop="name">${document.description}</span></div>
+
+                    <div class="detail-list-grid">
+                        <div class="grid-row">
+                            <div class="column column--reset">
+                                <dl class="detail-list">
+                                    <#if hasLocalAddress>
+                                    <dt class="detail-list__key">Address:</dt>
+                                    <dd class="detail-list__value" data-uipath="document.locAddress">
+                                        <#if document.locAddress.buildinglocation?has_content >
+                                        <span itemprop="location">${document.locAddress.buildinglocation}</span>,</#if> <span itemprop="number">${document.locAddress.buildingname}</span><#if document.locAddress.street?has_content >
+                                        <span itemprop="street">${document.locAddress.street}</span></#if><#if document.locAddress.area?has_content>, <span itemprop="area">${document.locAddress.area}</span></#if><#if document.locAddress.city?has_content>, <span itemprop="city">${document.locAddress.city}</span></#if><#if document.locAddress.county?has_content>, <span itemprop="county">${document.locAddress.county}</span></#if>, <span itemprop="country">${document.locAddress.country}</span>, <span itemprop="postcode">${document.locAddress.postalCode}</span>&nbsp;&nbsp;
+                                        <#if document.locAddress.mapLink?has_content>
+                                        <a href="${document.locAddress.mapLink}"
+                                            title="openstreetmaplink" target="_blank"><span itemprop="subjectof" itemscope itemtype="https://schema.org/webapi">Open in streetmap</span>
```

Comment:
Update title attribute to something more user friendly for example: Open StreetMap link in new window.  Could do the same for others if applicable